bin/tessel-2.js

```diff
@@ -324,11 +324,40 @@ makeCommand('ap')
 
 makeCommand('root')
   .callback(function(opts) {
+    opts.usb = false;
```

Comment:
Can you move this to `controller`? I already marked [`opts.lan = true`](https://github.com/tessel/t2-cli/blob/0772e4558bf264fb48389ed85953b103dd6c7059/lib/controller.js#L713) and we can be more explicit about USB there as well.